provider/resources.go

```diff
@@ -133,5 +135,30 @@ func Provider() tfbridge.ProviderInfo {
 	return prov
 }
 
+func docEditRules(defaults []tfbridge.DocsEdit) []tfbridge.DocsEdit {
+	return append(
+		defaults,
+		skipText,
+	)
+}
+
+var regexpsToSkip = []*regexp.Regexp{
+	// Matches TF specific warnings of the format `!> warning text etc...`
+	regexp.MustCompile(`!>(?s:.)*?\n\n`),
```

Comment:
Why don't we translate these to hugo warning/note blocks? We can support warning/note blocks in our docs.

This feels like the change could be upstreamed into the bridge, since lots of providers have warning/note/info blocks.

Reply:
for sure! We can do that in general and should. But [these warnings are TF specific](https://registry.terraform.io/providers/Venafi/venafi/latest/docs). Since this is the landing page, I'm doing a litte more cleanup than would be necessary for regular resource docs pages.